pkgs/applications/misc/mbrola/default.nix

```diff
@@ -0,0 +1,106 @@
+{ stdenv, fetchurl, unzip, bash, patchelf, stdenv_32bit, makeWrapper, gcc_multi }:
+
+let
+	TCTS="https://tcts.fpms.ac.be/synthesis/mbrola";
+
+	# debian provides manpage and libstrongexit.so
+	# read https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=856331;msg=5
+	deb = fetchurl {
+		url = "http://http.debian.net/debian/pool/non-free/m/mbrola/mbrola_3.01h+2-3.debian.tar.xz";
+		sha256 = "16m99l5xscq20vishdk4y9svsmzfdw3f54ifjdhikgpsw2blbnq0";
+	};
+
+	binaries = rec {
+	      i686-linux = fetchurl  {
+		    url = "${TCTS}/bin/pclinux/mbr301h.zip";
+		    sha256 = "0vlxb2jpvps4nbgm9dfglik0973bf4fpr0v8rrpj08y8jynjyh6z";
+	      };
+	      x86_64-linux = i686-linux; # mbrola_AMD_Linux.zip is outofdate, we have to use 32bits
+	      powerpc-linux = i686-linux;
+    };
+
+    voicefr1 = fetchurl {
+		url = "${TCTS}/dba/fr1/fr1-990204.zip";
+		sha256 = "0c9qza71l6in4c6rgl5pqhy7x4as9fcfyb6sxqy3rxaf547w7d4f";
+	};
+
+    voicefr2 = fetchurl {
+		url = "${TCTS}/dba/fr2/fr2-980806.zip";
+		sha256 = "01qk4wq20gap1inj43igy1llg3rq685vdgrvsflhvl3fcxhn70rv";
+	};
+in
+
+stdenv.mkDerivation rec {
+
+	pname = "mbrola";
+	name = "${pname}-${version}";
+	version = "3.0.1h";
+	
+	src = deb;
+	binary = binaries."${stdenv.system}" or (throw "unsupported system: ${stdenv.system}");
+
+	buildInputs = [ unzip ];
+	nativeBuildInputs = [ patchelf makeWrapper gcc_multi ];
+	propagatedBuildInputs = [ bash ];
```

Comment:
This will probably not do the thing you want. It basically adds `bash` to `buildInputs` of all packages that have `mbrola` in `buildInputs`.